packages/design-system/src/components/OcResourceIcon/OcResourceIcon.vue

```diff
@@ -41,30 +44,43 @@ export default {
     size: {
       type: String,
       default: 'large',
-      validator: (value) => {
-        return value.match(/(xsmall|small|medium|large|xlarge|xxlarge|xxxlarge)/)
+      validator: (value: string): boolean => {
+        return AVAILABLE_SIZES.some((e) => e === value)
       }
     }
   },
   computed: {
     iconName() {
+      if (this.isSpace) return defaultSpaceIcon
```

Comment:
I know it was like this already, but we decided to not use one-line-if-blocks in the project because they are somewhat ugly to read.

Reply:
.... and because they make debugging more cumbersome and make diffs harder to read when changed later ;-)

Reply:
So just plain old

```js
if (true) {
  return "something"
}
```

then, just for clarification?

And what about `return "something" ? condition : "fallback"`, are those types of statements allowed (since I see them used in the same component also)?

Reply:
> I really like that we have a space resource icon now! Should we maybe use the same color as in the sidebar and the projects view? A brighter blue is being used there.
> 
> ![image](https://user-images.githubusercontent.com/50302941/207822976-6b4bca86-6359-4981-a992-10fef290cbbf.png)

Great eye! I've updated it and would need from @tbsbdr if a ResourceIcon of type "space" should have a more distinct color down the road - for now it'll be the default "passive swatch" color, as it already is in other places right now

Reply:
`return "something" ? condition : "fallback"` or `const someVariable = "something" ? condition : "fallback"` is fine. We just don't want to have the "one-line-return/continue".